model/status_board/v1/status_type.model

```diff
@@ -25,6 +25,9 @@ class Status {
   // The associated service ID.
   Service Service
 
+  // Additional Service related data.
+  ServiceInfo Interface
```

Comment:
Why is this an interface? Why not create a named struct with the necessary attributes in it?
```
class ServiceInfo {
  Fullname string
  StatusType string
}
```